### DISCUSSION TITLE
convert 7.1.0-1 does corrupt XML metadata

### DISCUSSION THREAD

**Discussion by freakout42** [ImageMagick/ImageMagick#3857] [Development]
convert 7.1.0-1 does corrupt XML metadata on some of my images. convert is self-compiled 7.0.x does not have the issue with the same build-environment - here is a log of the problem - the image is attached. Thanks for help
`
+ /opt/imagic/bin/convert --version
Version: ImageMagick 7.1.0-1 Q16 i686 2021-06-21 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP(4.0)
Delegates (built-in): bzlib fontconfig freetype jng jpeg png tiff xml zlib

+ /opt/imagic/bin/convert -size 1600x1060 -depth 8 d1508-pbonsi-2645.jpg -density 72 -quality 72.25 -scale 1600x1060 -unsharp 0x0.39+0.3+0.05 t.jpg

+ /opt/imagic/bin/identify t.jpg
t.jpg JPEG 1600x1060 1600x1060+0+0 8-bit sRGB 523259B 0.000u 0:00.000
identify: CorruptImageProfile `t.jpg' (XMP) @ warning/profile.c/ValidateXMPProfile/1920.

+ exiftool t.jpg
---- ExifTool ----
ExifTool Version Number         : 12.26
Warning                         : XMP format error (no closing tag for ±Ã)
Warning                         : [minor] Empty XMP
...
`

![d1508-pbonsi-2645](https://user-images.githubusercontent.com/3446978/123092266-9a74e880-d42a-11eb-807d-543daddb06bc.jpg)

**Comment by dlemstra** [comment]
Probably related to: https://github.com/ImageMagick/ImageMagick6/issues/159. Can you try this again with the latest version of ImageMagick @freakout42?

**Reply by freakout42** [reply]
Dear dlemstra - 7.1.0-2 is now Ok - all my tests run with it Ok. Thank you for your help and for ImageMagick.
